apps/client/ui/MoviesList/MoviesList.tsx

```diff
@@ -1,34 +1,31 @@
 import type { ComponentPropsWithoutRef, FC } from 'react';
-import { useState, useEffect } from 'react';
 import clsx from 'clsx';
 import axios from 'axios';
 import { useSelector } from 'react-redux';
+import { useQuery } from '@tanstack/react-query';
 
 import MovieTile from '@/ui/MovieTile/MovieTile';
 import { RootState } from '@/store/store';
 
 type MoviesListProps = ComponentPropsWithoutRef<'ul'>;
 
 const MoviesList: FC<MoviesListProps> = ({ className, ...props }) => {
-  const [movies, setMovies] = useState([]);
-  const [loading, setLoading] = useState(true);
   const isGrid = useSelector((state: RootState) => state.main.isGrid);
 
-  useEffect(() => {
-    const fetchData = async () => {
-      try {
-        const response = await axios.get('http://localhost:3001/top-rated');
-        setMovies(response.data.data);
-      } catch (error) {
-        console.error(error);
-      } finally {
-        setLoading(false);
-      }
-    };
-    fetchData();
-  }, []);
+  const {
+    isPending,
+    error,
+    data: movies,
+  } = useQuery({
+    queryKey: ['movies'],
+    queryFn: async () =>
+      await axios
```

Comment:
lets use `@/api` instead of plain `axios` 👌 then TS will tell you about the change with image - `backdrop_image` -> `poster_image` 😄

Reply:
I see you made the change in second PR 💪 no need to do it here 😄